lib/contentTypeParser.js

```diff
@@ -279,4 +279,8 @@ module.exports.helpers = {
   addContentTypeParser,
   hasContentTypeParser
 }
+module.exports.defaultParsers = {
+  getDefaultJsonParser,
```

Comment:
```suggestion
  defaultJsonParser,
```

Reply:
defaultJsonParser is a inner function which needs proto poisoning and constructor poisoning from getDefaultJsonParser. Would you want me to bring the function out and modify getDefaultJsonParserto call the outer defaultJsonParser function ?

Reply:
I have made a small tweak to achieve this